modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -249,18 +269,14 @@ export default class Tile3DLayer extends CompositeLayer {
         coordinateOrigin: cartographicOrigin,
         modelMatrix,
 
-        getColor: constantRGBA || this.props.color,
-        pickable: true,
-        numInstances: pointCount,
-        opacity: 1.0,
-        pointSize: 2
-      })
+        getColor: constantRGBA || getColor,
+        numInstances: pointCount
```

Comment:
Do we now need to specify pointCount two times, both in `numinstances` and `data.header.vertexCount`?

Reply:
`numinstances` should not be needed